### DISCUSSION TITLE
Since I upgraded to Next 13, using jose jwtVerify inside the nextjs middleware is causing an error called ERR_JWS_INVALID.

### DISCUSSION THREAD

**Discussion by gmotta-dev** [vercel/next.js#49423] [Help] [answered]
### Summary

when using this line of code

` const userToken = req?.cookies.get('user_token')`
` const decoded = await verify(userToken, process.env.JWT_KEY)`
`console.log(decoded)`

Returns this 
` [JWSInvalid: Compact JWS must be a string or Uint8Array] {`
`  code: 'ERR_JWS_INVALID',`
`  name: 'JWSInvalid'`
`}`

If instead of using userToken inside the verify function, like this: 
` const decoded = await verify(userToken.value, process.env.JWT_KEY)`

It works just fine, but it was not like that when I was using nextjs 12

**Comment by icyJoseph** [comment, marked as answer]
I would just like to make sure that, the issue is not related to https://nextjs.org/docs/messages/middleware-upgrade-guide#cookies-api-revamped, where the cookies API was reworked to return an object instead of a string.

Could you confirm?
